chain-evm/Cargo.toml

```diff
@@ -22,7 +22,8 @@ hex = { version = "0.4", default-features = false, features = ["alloc"] }
 byte-slice-cast = { version = "1.0", default-features = false }
 thiserror = "1.0"
 quickcheck = { version = "0.9", optional = true }
-evm = { version = "0.34.0" }
+evm = { version = "0.35.0" }
+ethereum = { version = "0.12.0" }
```

Comment:
Why this dependency is needed ?

Reply:
It's already in #786, and now I have removed it from this branch.